lib/main_common.pm

```diff
@@ -157,8 +157,14 @@ sub load_autoyast_tests {
     else {
         loadtest("autoyast/repos");
         loadtest("autoyast/clone");
+        if (get_var('SP3ORLATER') && check_var('FILESYSTEM', 'btrfs')) {
+            loadtest "autoyast/verify_autoinst_btrfs";
+        }
         loadtest("autoyast/logs");
     }
+    if (get_var('SP3ORLATER') && check_var('FILESYSTEM', 'btrfs')) {
```

Comment:
as discussed in person, you were right, I was wrong. everything fine :-) Make sure the tests are triggered accordingly on the production instance